docs/create_your_first_nn.md

```diff
@@ -0,0 +1,118 @@
+In this tutorial, you'll learn how you can create your first neural network with Kotlin DL. We'll guide you through the 
+basic building blocks and concepts you'll need to get started with Deep Learning. 
+This tutorial does not require any previous experience with Deep Learning. 
+
+## Image Classification
+
+For the purposes of this tutorial, we'll be using the [Fashion-MNIST dataset](https://github.com/zalandoresearch/fashion-mnist)
+which you can think of a "Hello World" dataset in Deep Learning. 
+This dataset contains 70000 images that represent 10 different categories of clothing items. 
+Each example is a 28x28 grayscale image, associated with a single label from 10 classes. Here are the labels and how 
+they map to actual classes: 
+
+| Label        | Class           | 
+| ------------- |:-------------:| 
+| 0      | T-shirt/top | 
+| 1      | Trouser |
+| 2      | Pullover |
+| 3      | Dress |
+| 4      | Coat |
+| 5      | Sandal |
+| 6      | Shirt |
+| 7      | Sneaker |
+| 8      | Bag |
+| 9      | Ankle boot |
+
+And this is what the images themselves look like: 
+![](images/fashion-mnist-sprite.png)
+
+In this tutorial we are going to build and train a neural network that will be able to classify images like these 
+into given categories. 
+
+## Building a Neural Network
+
+Before our neural network can learn from the data, we need to define the structure of the neural network itself. 
+The basic building block of a neural network is a **layer**. To define a neural network, you need to describe 
+what layers it should consist of. The goal of those layers is to capture some data representation aspects during 
+the training phase. With enough data, layers, and training time, neural networks are capable of capturing enough 
+complexity to result in great performance on many tasks. 
+
+Here, our neural network will consist of a few simple layers in a sequence:  
+
+```kotlin
+private val model = Sequential.of(
+    Input(28,28),
```

Comment:
Looks like it should be Input (28, 28, 1).  We should be careful about input data shapes

Reply:
1 - is number of channels (1 for greyscale or 3 for typical png image)

Maybe I need to add a warning about that